src/actions/purchaseDomains.ts

```diff
@@ -47,10 +71,30 @@ export const purchaseDomains = async (
     `Not enough funds given for purchase of ${count} domains`
   );
 
+  const sellerWallet = await contract.sellerWallet();
+
+  // If using a Gnosis safe as the seller wallet you must
+  // provide the implementation address for the tx accessList
+  const sellerWalletImpl = process.env.SELLER_WALLET_IMPL;
```

Comment:
We can't be reading from env variables in our SDK. This is going to be running in a user's browser. You need to get this some other way.

Reply:
There is a way to get the implementation address on-chain at the time this function is called. It's somewhat advanced but can be accomplished with a single smart contract call.

Reply:
Will figure this out